<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Recursos.qrc">
    <normaloff>:/Recurso/checklist.png</normaloff>:/Recurso/checklist.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(62, 147, 147);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(142, 195, 195);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Nirmala UI</family>
              <pointsize>24</pointsize>
              <underline>true</underline>
             </font>
            </property>
            <property name="text">
             <string>Fast Work</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Registra tus tareas facilmente ! </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Segoe UI Semibold</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>tu App de registro</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Nombre del pendiente:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="innombre">
            <property name="toolTip">
             <string/>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(142, 195, 195);</string>
            </property>
            <property name="placeholderText">
             <string>Ingrese su pendiente</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Fecha limite:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="infecha">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(142, 195, 195);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Priorida:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="inalta">
            <property name="toolTip">
             <string>prioridad alta</string>
            </property>
            <property name="text">
             <string>Alta</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="inmedia">
            <property name="toolTip">
             <string>prioridad media</string>
            </property>
            <property name="text">
             <string>Media</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="inbaja">
            <property name="toolTip">
             <string>prioridad baja </string>
            </property>
            <property name="text">
             <string>Baja</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="inProducto">
            <property name="toolTip">
             <string>tipo de actividad</string>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(142, 195, 195);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Tipo: </string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PBlistar">
         <property name="toolTip">
          <string>pulsa para a√±adir</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(142, 195, 195);</string>
         </property>
         <property name="text">
          <string>Listar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(142, 195, 195);</string>
      </property>
      <property name="title">
       <string>Previsualizacion</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableWidget" name="outDetalle">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(142, 195, 195);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="infinalizar"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSalir"/>
   <addaction name="infinalizar"/>
   <addaction name="actionAcerca_de"/>
  </widget>
  <action name="actionAcerca_de">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Recurso/about (2).png</normaloff>:/Recurso/about (2).png</iconset>
   </property>
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
  <action name="infinalizar">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Recurso/delete.png</normaloff>:/Recurso/delete.png</iconset>
   </property>
   <property name="text">
    <string>Finalizar</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Recurso/logout (2).png</normaloff>:/Recurso/logout (2).png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
